fe/fe-core/src/main/java/com/starrocks/clone/TabletSchedCtx.java

```diff
@@ -1293,31 +1293,25 @@ public boolean checkPrivForCurrUser(UserIdentity currentUser) {
             return true;
         }
 
-        Locker locker = new Locker();
-        try {
-            locker.lockDatabase(db.getId(), LockType.READ);
-            Table table = GlobalStateMgr.getCurrentState().getLocalMetastore().getTable(db.getId(), tblId);
-            if (table == null) {
+        Table table = db.getTable(tblId);
+        if (table == null) {
+            return true;
+        } else {
+            // if user has 'OPERATE' privilege, can see this tablet, for backward compatibility
```

Comment:
```suggestion
            return true;
        }
        // if user has 'OPERATE' privilege, can see this tablet, for backward compatibility
```
It seems that there is no need to have the `else` indent.